examples/jsm/interactive/InteractiveGroup.js

```diff
@@ -7,97 +7,147 @@ import {
+
+		this.camera = camera;
+		this.element = renderer.domElement;
+
+		this.element.addEventListener( 'pointerdown', this._onPointerEvent );
+		this.element.addEventListener( 'pointerup', this._onPointerEvent );
+		this.element.addEventListener( 'pointermove', this._onPointerEvent );
+		this.element.addEventListener( 'mousedown', this._onPointerEvent );
+		this.element.addEventListener( 'mouseup', this._onPointerEvent );
+		this.element.addEventListener( 'mousemove', this._onPointerEvent );
+		this.element.addEventListener( 'click', this._onPointerEvent );
 
 	}
 
-	listenToXRControllerEvents( controller ) {
+	disconnectionPointerEvents() {
+
+		this.element?.removeEventListener( 'pointerdown', this._onPointerEvent );
+		this.element?.removeEventListener( 'pointerup', this._onPointerEvent );
+		this.element?.removeEventListener( 'pointermove', this._onPointerEvent );
+		this.element?.removeEventListener( 'mousedown', this._onPointerEvent );
+		this.element?.removeEventListener( 'mouseup', this._onPointerEvent );
+		this.element?.removeEventListener( 'mousemove', this._onPointerEvent );
+		this.element?.removeEventListener( 'click', this._onPointerEvent );
+
+	}
 
-		const scope = this;
+	disconnect() {
 
-		// TODO: Dispatch pointerevents too
+		this.disconnectionPointerEvents();
+		this.disconnectXrControllerEvents();
 
-		const events = {
-			'move': 'mousemove',
-			'select': 'click',
-			'selectstart': 'mousedown',
-			'selectend': 'mouseup'
-		};
+		this.camera = undefined;
+		this.controllers = [];
+		this.element = undefined;
 
-		function onXRControllerEvent( event ) {
+	}
 
-			const controller = event.target;
+	onXRControllerEvent( event ) {
 
-			_raycaster.setFromXRController( controller );
+		const controller = event.target;
 
-			const intersections = _raycaster.intersectObjects( scope.children, false );
+		_raycaster.setFromXRController( controller );
 
-			if ( intersections.length > 0 ) {
+		const intersections = _raycaster.intersectObjects( this.children, false );
 
-				const intersection = intersections[ 0 ];
+		if ( intersections.length > 0 ) {
 
-				const object = intersection.object;
-				const uv = intersection.uv;
+			const intersection = intersections[ 0 ];
 
-				_event.type = events[ event.type ];
-				_event.data.set( uv.x, 1 - uv.y );
+			const object = intersection.object;
+			const uv = intersection.uv;
 
-				object.dispatchEvent( _event );
+			_event.type = events[ event.type ];
+			_event.data.set( uv.x, 1 - uv.y );
 
-			}
+			object.dispatchEvent( _event );
 
 		}
 
-		controller.addEventListener( 'move', onXRControllerEvent );
-		controller.addEventListener( 'select', onXRControllerEvent );
-		controller.addEventListener( 'selectstart', onXRControllerEvent );
-		controller.addEventListener( 'selectend', onXRControllerEvent );
+	}
+
+	listenToXRControllerEvents( controller ) {
+
+		this.controllers.push( controller );
+		controller.addEventListener( 'move', this._onXRControllerEvent );
```

Comment:
I ported this over, but I don't think `move` is an actual XR event? It's not listed on the MDN https://developer.mozilla.org/en-US/docs/Web/API/XRSession/select_event

Reply:
Never mind, I missed that this is a synthetic event that the threejs controller itself dispatches:

https://github.com/mrdoob/three.js/blob/fcde431ddf191173491d3e604788ba75d3b43e18/src/renderers/webxr/WebXRController.js#L290